content/systems/filecoin_mining/storage_mining/mining_cycle/index.md

```diff
@@ -8,22 +8,24 @@ dashboardTests: 0
 
 # Storage Mining Cycle
 
-Block miners should constantly be performing Proofs of SpaceTime using [Election PoSt](election_post), and checking the outputted partial tickets to run [Leader Election](expected_consensus#secret-leader-election) and determine whether they can propose a block at each epoch. Epochs are currently set to take around X seconds, in order to account for election PoSt and network propagation around the world. The details of the mining cycle are defined here.
+Block miners should constantly be performing Proofs of SpaceTime using [Election PoSt](election_post), and checking the outputted partial tickets to run [Leader Election](expected_consensus#secret-leader-election) and determine whether they can propose a block at each epoch. Epochs are currently set to take 30 seconds, in order to account for election PoSt and network propagation around the world.
```

Comment:
We need to remove all the reference to Election Post here as well.

I would write something like: 

"Block miners should constantly, at each epoch, be checking if they win the leader election protocol [Leader Election](expected_consensus#secret-leader-election) and in case hey are elected, determine whether they can propose a block running the Winning PoSt. Epochs are currently set to take 30 seconds, in order to account for Winning PoSt and network propagation around the world.